exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/DispatcherSchema.java

```diff
@@ -36,8 +36,8 @@ public DispatcherSchema(View view) {
   /**
    * Returns a map of service instance specifications of started services indexed by their names.
    */
-  public MapIndexProxy<String, InstanceSpec> serviceInstances() {
-    return MapIndexProxy.newInstance("core.dispatcher.service_instances", view,
-        string(), protobuf(InstanceSpec.class));
+  public ProofMapIndexProxy<String, InstanceState> serviceInstances() {
+    return ProofMapIndexProxy.newInstance("dispatcher_instances", view,
```

Comment:
Again ProofMapIndex. I hope the name is correct, the tests for this method are disabled.

Reply:
com/exonum/binding/test/TimeSchemaProxyIntegrationTest.java:52 — got to be enabled then.

Reply:
Won't work though till hashing is done.

Reply:
Yep, I tried to enable disabled tests (also a lot of them in TestTestKit.java) - all of them are not working at the moment (due to hashing restricitons)